auth/memberships.go

```
@@ -175,11 +174,23 @@ func (svc service) ListOrgMemberships(ctx context.Context, token string, orgID s
 			return OrgMembershipsPage{}, err
 		}
 
-		for _, user := range page.Users {
+		userEmailByID := make(map[string]string, len(page.Users))
+		for _, u := range page.Users {
+			userEmailByID[u.Id] = u.Email
+		}
+
+		for _, m := range omp.OrgMemberships {
+			email, ok := userEmailByID[m.MemberID]
+			if !ok {
+				continue
```

It would be better to return the `OrgMembeship`with empty email than failing silently